src/app/hooks/useScreensize.ts

```diff
@@ -0,0 +1,11 @@
+import useMediaQuery from '@mui/material/useMediaQuery'
+import { useTheme } from '@mui/material/styles'
+
+export const useScreenSize = () => {
+  const theme = useTheme()
+  return {
+    isMobile: useMediaQuery(theme.breakpoints.down('sm')),
+    isTablet: useMediaQuery(theme.breakpoints.down('md')),
```

Comment:
This seems wrong. Should be between mobile and desktop

Reply:
After discussing the same issue with @lubej, I have opened https://github.com/oasisprotocol/explorer/issues/555.